src/Marker.svelte

```diff
@@ -35,6 +35,11 @@
 
 	  return () => marker.remove()
 	})
+	
+	afterUpdate(() => {
```

Comment:
I would use a reactive property for this: `$:`

Reply:
Done.. Removed it.